### ISSUE TITLE
[Bug] Yo-Kai Watch 1 flipped/gitched text frames

### ISSUE THREAD

**Issue by ubitux** [Ryujinx/Ryujinx#5255]
### Description of the issue

One of the first text frame in the game is flipped (but the issue does not seem to happen later in the game). I got these rendered glitched in one of my run test too (like, filled with another texture, maybe that was random memory). The logs is filled with Vulkan validation errors too.

![ryujinx_capture_2023-06-07_00-13-02](https://github.com/Ryujinx/Ryujinx/assets/34467/70d65a7b-d890-429c-a007-32d92c9fadc3)

### Reproduction steps

1. Start a new game
2. Create the character
3. Follow the dialogues up until you have control of the character
4. Talk to your mom, then leave the house
5. Follow the red arrow into the park
6. Go to the tree on the right, and press A
7. capture the cicada

You should see something similar to the screenshot above.

### Log file

The log contains all the debug enabled so it is compressed: [yo-kai-watch-log.zip](https://github.com/Ryujinx/Ryujinx/files/11670785/yo-kai-watch-log.zip)

### OS

Linux

### Ryujinx version

1.1.869

### CPU

i7-12700

### GPU

Radeon RX 6400

### RAM

64G

**Comment by ubitux** [comment]
Filtering out all the validation related to `srcAccessMask`, `dstAccessMask`, and `pImageMemoryBarriers` leaves us with the following warnings:

These ×4:
```
Validation Error: [ VUID-vkDestroyBuffer-buffer-00922 ] Object 0: handle = 0x7dc900c75f20, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0xe4549c11 | Cannot call vkDestroyBuffer on VkBuffer 0xb0c099000000b72b[] that is currently in use by a command buffer. The Vulkan spec states: All submitted commands that refer to buffer, either directly or via a VkBufferView, must have completed execution (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkDestroyBuffer-buffer-00922)
```

These ×3:
```
Validation Error: [ VUID-vkDestroyImageView-imageView-01026 ] Object 0: handle = 0x7dc900c75f20, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0x63ac21f0 | Cannot call vkDestroyImageView on VkImageView 0xa17c25000000a0c6[] that is currently in use by a command buffer. The Vulkan spec states: All submitted commands that refer to imageView must have completed execution (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkDestroyImageView-imageView-01026)
```

The first 4 in particular seem to happen around the time of the glitch.

I'm not familiar with the codebase so I'm unlikely to be able to provide much more information currently.

**Comment by gdkchan** [comment]
Fixed by #5943.
